### ISSUE TITLE
[A11y] Mobile Collection Landing Facets

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#2007]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

No

### Issue Description

Mobile Collection Landing: https://staging.oregondigital.org/collections/gwilliams

As a non logged in user, I tried to use the facets and when I chose one of the facet links for example Date > 2012. It took me to a login page. Stated that I was not authorized to access this page. It also gave me the same error when I tried Media > Images.

Should this be available to a non logged in user if they are not able to use it.

### Can anybody QA or does this need an expert?

Anybody

**Comment by straleyb** [comment]
QA: Collection show advanced search facets should behave like advanced search with the properly configured facets for said collection.

**Comment by azle** [comment]
Updated 12/26/2024
@jsimic, just to have another set of eyes. I feel like this is no longer valid. The filter doesnt have dates anymore and instead has decades I chose one and it worked just fine on mobile.

I used this link:
https://staging.oregondigital.org/collections/gwilliams

I also wanted to check with you that we are no longer using dates and it is replaced with decades.
